src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randomColor = new ColorSupplier();
+        String color = randomColor.getRandomColor();
+        int number = new Random().nextInt(100);
+        return new Ball(color, number);
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. And lets move number 100 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names